app/controllers/publishers/vacancies_controller.rb

```diff
@@ -11,10 +11,18 @@ def show
 
   def create_or_copy; end
 
-  def select_a_job_for_copying; end
+  def select_a_job_for_copying
+    @form = Publishers::JobListing::SelectAJobForCopyingForm.new
+  end
 
   def redirect_to_copy_job
```

Comment:
I am not 100% sure about the name of this method, but cannot think a better one for now